src/components/interface/AudioPlay.jsx

```diff
@@ -33,20 +89,23 @@ class AudioPlayer extends React.Component {
         <div style={{ alignSelf: 'center', flex: 3 }}>
           {fileName}
         </div>
-        <div style={{ alignSelf: 'center', flex: 2 }}>
+        <div style={{ alignSelf: 'center', flex: 0 }}>
           {time}
         </div>
-        <div>
-          <PlayCircleOutlineIcon style={{ float: 'right', marginLeft: '25px' }} color="primary" />
+        <div style={{ flex: 0 }}>
+          {
+            <Button disabled={audioFile == undefined ? true : false} type='transparent' onClick={this.handleClick}>
```

Comment:
Just to check my logic here: isn't `disabled={audioFile == undefined ? true :  false}` the same as `disabled={audioFile == undefined}` ?

Also: preferred to use `===`

Reply:
Also can we alphabetize these props?

Reply:
> Just to check my logic here: isn't `disabled={audioFile == undefined ? true : false}` the same as `disabled={audioFile == undefined}` ?

Yup! It's changed now to `disabled={audioFile == undefined}`